spec/cc/engine/analyzers/ruby/main_spec.rb

```diff
@@ -197,5 +235,18 @@ def self.from_remediation_amount(amount)
     def engine_conf
       EngineConfig.new({})
     end
+
+    def filtered_engine_conf *patterns
+      EngineConfig.new({
```

Comment:
Intentional, yes. Good? probably not. It's how emacs does it by default (more like lisp indentation).